src/containers/DashboardWrapper/UpgradeTavlaBanner/styles.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../../assets/styles/import.scss';
+
+.banner {
+    background: $colors-validation-canary;
+    width: 100%;
+    height: 37px;
+    text-align: center;
+    line-height: 21px;
```

Comment:
Du har 2 `line-height`'s i `banner`. Kanskje også bruke partall i stedet for oddetall? Er sjeldent at det faktisk trenger å være et oddetall 👍